decidim-core/app/commands/decidim/amendable/accept.rb

```diff
@@ -68,7 +68,7 @@ def amendable_attributes
       end
 
       def notify_amendable_and_emendation_authors_and_followers
-        affected_users = @emendation.authors + @amendable.authors
+        affected_users = @emendation.authors + @amendable.notifiable_identities
```

Comment:
Should we ensure those are unique? (same for other lines in this PR)

Reply:
Of course. Thanks!